shell.nix

```diff
@@ -28,6 +28,7 @@ let
 
     tools = {
       cabal = "3.2.0.0";
+      fourmolu = "0.3.0.0";
```

Comment:
added this temporarily because `ormolu` annoyingly wipes out multi-line haddock comments. Fourmolu has an option to avoid this.